docs/ops.html

```diff
@@ -62,8 +62,8 @@
   <pre class="brush: bash;">
   &gt; bin/kafka-topics.sh --zookeeper zk_host:port/chroot --delete --topic my_topic_name
   </pre>
-  Topic deletion option is disabled by default. To enable it set the server config
-    <pre class="brush: text;">delete.topic.enable=true</pre>
+  Since 1.0.0, Topic deletion option is enabled by default. To disable it set the server config
+    <pre class="brush: text;">delete.topic.enable=false</pre>
```

Comment:
We should probably say that it's enabled by default unless the following config is set. People reading this don't want to disable it.

Reply:
@ijuma update the doc.  pl check if this is ok

Reply:
@ijuma Can you pls merge this minor change

Reply:
@ijuma  This doc change needs to updated on 1.0.0 docs

Reply:
cc @guozhangwang